packages/javascript/bh-shared-ui/src/components/CreateUserForm/CreateUserForm.tsx

```diff
@@ -46,33 +46,22 @@ const CreateUserForm: React.FC<{
         handleSubmit,
         setValue,
         formState: { errors },
-    } = useForm({
-        defaultValues: {
-            emailAddress: '',
-            principal: '',
-            firstName: '',
-            lastName: '',
-            password: '',
-            needsPasswordReset: false,
-            SAMLProviderId: '',
-            roles: [1],
-        },
```

Comment:
I just wanted to make sure that deleting this doesn't cause weird behavior on `reset()`

Reply:
Just tested creating a user and then re-opening the dialog & filling out the create user form, closing the form, and re-opening it. Both set the form back to the default values